open-metadata-resources/open-metadata-labs/ui-labs/ui-asset-search.ipynb

```diff
@@ -0,0 +1,290 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> **Callie Quartile:** Data Scientist"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up\n",
+    "\n",
+    "Coco Pharmaceuticals make widespread use of ODPi Egeria for tracking and managing their data and related assets.\n",
+    "Figure 1 below shows their metadata servers and the Open Metadata and Governance (OMAG) Server Platforms that are hosting them.  Each metadata server supports a department in the organization.  The servers are distributed across the platform to even out the workload.  Servers can be moved to a different platform if needed.\n",
+    "\n",
+    "![Figure 1](../images/coco-pharmaceuticals-systems-omag-server-platforms.png)\n",
+    "> **Figure 1:** Coco Pharmaceuticals' OMAG Server Platforms\n",
+    "\n",
+    "The code below sets up the network addresses for the three platforms.  This varies depending on whether you are running them locally, through **docker-compose** or on **kubernetes**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run ../common/environment-check.ipynb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "Callie is using the data lake operations metadata server called `cocoMDS1`. This server is hosted on the Data Lake OMAG Server Platform.\n",
+    "\n",
+    "Check that `cocoMDS1` is running.  \n",
+    "\n",
+    "If any of the platforms are not running, follow [this link to set up and run the platform](https://egeria.odpi.org/open-metadata-resources/open-metadata-labs/).  If any server is reporting that it is not configured then\n",
+    "run the steps in the **[Server Configuration](../egeria-server-config.ipynb)** lab to configure\n",
+    "the servers.  Then re-run the previous step to ensure all of the servers are started."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "## Exercise 1\n",
+    "\n",
+    "### Importing IGC events\n",
+    "\n",
+    "IBM InfoSphere Information Governance Catalog (IGC) is a tool that allows you to create, manage and share a common business language, document and enact policies and rules and track data lineage.\n",
+    "\n",
+    "For this part of the exercises, we have prepared a set of events for Data Engine OMAS in `cocoMDS1` to pick up. The precise process on how these events have been created is more comvoluted, and is out of scope of this lab."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Install kafka-python"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will send the IGC events through the kafka bus. The `kafka-python` package allows us to interact with kafka directly with Python."
```

Comment:
This the only way we have at the moment to load data. Those events is this file [https://github.com/odpi/egeria/blob/master/open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/payloads/omrs-cohort-ibm-igc.txt](url)

Yes they are coming from IGC but are mapped to the Open metadata Types so they are result from the mapper.

We cannot load from an archive or use an API ( at least I don't know if we have that option) since we need to produce events to fill egeria graph which is the only one in the cohort with this test scenario.

Reply:
I use an API to populate instance data - in my case it's using the OMRS MetadataCollection APIs via REST; I would imagine other people might be more comfortable using an OMAS REST API.

Reply:
I am concerned due to a) Reference to a proprietary solution b) the opaqueness of using an internal messaging format to the end user, and setting a bad precedent that it is a supported mechanism.

At a minimum I think we need to discuss this on a call, but as per Graham's comment above, I think it should be possible, clear, and setting the correct precedent, by showing how to do things using the APIs.

Reply:
@planetf1 I have updated the notebook to create simple csv files using asset owner 
7abc4e1. Could you have another look if it is good enough?

Reply:
Thanks. will do - though could be Mon now sorry, just saw...

Reply:
@planetf1 Thanks.